src/stars.css

```
@@ -0,0 +1,45 @@
+.card__review {
+  display: flex;
+  text-align: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--1 :nth-child(n+5),
+.stars--2 :nth-child(n+4),
+.stars--3 :nth-child(n+3),
+.stars--4 :nth-child(n+2),
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

```suggestion
.stars--5 :nth-child(n) {
  background-image: url(images/star-active.svg);
```
do not repeat yourself